src/utils/qParamHelper.ts

```diff
@@ -204,11 +204,7 @@ export const removeQParamItem = (id: string, value: TQParamValue, operator: IQPa
 
 /**
  * @param qParam - RSQL string: filename=like="%te%t%";status!=CANCELLED
- * @returns Object representing individual RSQL items deeply parsed:
- * {
- *   name: ['"%te%t%"'],
- *   status: ['CANCELLED']
- * }
+ * @returns Object representing individual RSQL items deeply parsed along with logical operators
```

Comment:
Can you please keep the return example that corresponds to `filename=like="%te%t%";status!=CANCELLED` ?